packages/story-editor/src/components/panels/layer/groupLayer.js

```diff
@@ -111,8 +113,10 @@ function GroupLayer({ groupId }) {
       }
       layerName={name}
       handleClick={handleClick}
+      hasLayerHiddenIcon={isHidden}
       isSelected={isSelected}
       LayerIcon={GroupLayerIcon}
+      LayerVisibilityIcon={LayerVisibilityIcon}
```

Comment:
Q: Is there a benefit to passing this forward as a prop instead of using `hasLayerHiddenIcon` in `Layer` component to get the `LayerVisibilityIcon`?

Reply:
I think no, but I did the same thing as we did for locking with hope that it makes sense 😁

Reply:
Move both to reduce prop drilling without a reason? 😄

Reply:
Ok, so there is a reason to pass the Icon as a prop: the Layer component is used for both layer and group, for layer it's simple but for a group there is a condition, so we cannot hardcode the icon in the Layer component. Also, with the new layer visibility icon for the group (deduced from layers within the group) it makes sense to do it that way.